### ISSUE TITLE
JAWS + Chrome: Focus incorrectly moves to heading

### ISSUE THREAD

**Issue by turbosheep44** [FreedomScientific/VFO-standards-support#594]
### Summary

In some situations JAWS on Chrome incorrectly tells the user that a heading is being focused but Chrome shows a button being focused. Then, if the user presses Enter to active the button, nothing happens, effectively breaking functionality on the website.

### Expected result

JAWS should read the information about the selected button and the button should work correctly when Enter is pressed.

### Actual result

JAWS reads incorrect information about the focused element and the button does not work.

### Example

[Codepen Example](https://codepen.io/turboSheep/pen/qBPmjYN), follow the instructions given to replicate.

### Additional Information

The bug is *not* present on Firefox (with or without JAWS).
With JAWS disabled, the website works as expected on Chrome.
According to Chrome DevTools `document.activeElement` is the deselect button, even though JAWS says that the `h2` is selected.
This issue may be similar to #172.

#### JAWS version and build number
Version 2022.2110.36
 
#### Operating System and version
Windows 10 Version 10.0.19042

#### Browser and version
Chrome Version 96.0.4664.93 (Official Build) (64-bit)

**Comment by JAWS-test** [comment]
I suspect that this is not really a bug of JAWS, but an implementation problem in your example: an element with role=combobox should not contain buttons. A combobox is an element that contains only text (the current value). It is not intended that other interactive elements are nested in a combobox.

**Comment by JAWS-test** [comment]
I recommend to clarify on the one hand at https://github.com/w3c/aria/issues which child elements a combobox may contain and on the other hand to ask at https://github.com/w3c/aria-practices/issues how a multiselect combobox should be implemented (in my opinion there are no good ideas for this at the moment)

**Comment by stevefaulkner** [comment]
tested, can reproduce, works as expected in NVDA
